swagger.yaml

```diff
@@ -2472,6 +2472,44 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /boot-files:
+    get:
+      tags:
+        - installer
```

Comment:
Should probably be in a different handler.

Reply:
Moved to new handler